Changelog.md

```diff
@@ -7,6 +7,10 @@
 - Replace `pyCalverter` with `convertdate` (#536)
 - Remove unused `JalaliMixin`
 
+## v15.5.0 (2021-07-18)
```

Comment:
Please do not assign a release number or a new version, it'll be done by the maintainer.
You may remove this line, and make sure that the changelog item is associated with the "master" section.

Reply:
Updated changelog